go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kserve/modelmesh-runtime-adapter
 
-go 1.19
+go 1.21
```

Comment:
Did you run `go mod tidy -compat=1.21` to update the list of dependencies below? I would expect most of those to change.

Reply:
as the rest-proxy one, just go mod tidy, let me try.

Reply:
here, we have some changes, pushed.
thanks.